addons/pos_online_payment_self_order/static/src/self_order_service.js

```diff
@@ -11,20 +11,15 @@ patch(SelfOrder.prototype, {
             this.onlinePaymentStatus = status;
             this.paymentError = status === "fail";
 
+            const device = this.config.self_ordering_mode;
```

Comment:
We are not using `device` anywhere else so it doesn't make sense to store in a variable